src/hypothesis/searchstrategy/numbers.py

```diff
@@ -78,8 +78,8 @@ def distribution(random, n):
             else:
                 r &= (~sign_mask)
             return int_to_bytes(r, n)
-        byt = data.draw_bytes(size, distribution=distribution)
-        r = int_from_bytes(byt)
+        b = data.draw_bytes(size, distribution=distribution)
+        r = int_from_bytes(b)
```

Comment:
-1 to this. I think longer variable names are generally better: they usually provide a better description of the variable (certainly compared to a single letter), and they’re easier to grep for.

It’s not worth it to go through the code base and expunge all single-letter variable names, but I definitely wouldn’t introduce new ones.